packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.0+2
+
+* Add a dependency on kotlin-bom to align versions kotlin transitive dependencies.
```

Comment:
Is this missing an "of" in "versions kotlin"?

(also, "Kotlin")

Reply:
Yes it is 🤦. It was originally "to align Kotlin transitive dependencies" and I missed an "of" when adding "versions"